### PR TITLE
Update unattended installation doc and deprecate ister doc.

### LINKED ISSUE #1052: Convert the ister.py doc to clr-installer
Change the ister.py doc:
https://docs.01.org/clearlinux/latest/guides/clear/ister.html#

to use clr-installer.

### PR DESCRIPTION
**Pull request by bktan8**
Closes #1052

Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/get-started/install-configfile.rst [hunk 1]
```diff
@@ -3,10 +3,11 @@
...
 
-This page explains how to install |CL-ATTR| using the clr-installer tool
-with a configuration file. The configuration file (:file:`clr-installer.yaml`)
-can be reused to duplicate the same installation configuration on additional
-machines.
+In addition to the interactive GUI and text-based modes,
+:command:`clr-installer` also supports an unattended mode where you 
+simply provide it a YAML configuration file.  
+
+This guide shows you three examples of how to use its unattended mode.
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: This guide shows three examples of how to use the unattended mode.

File: source/get-started/install-configfile.rst [hunk 2]
```diff
@@ -15,67 +16,61 @@ machines.
...
+#. In the console window, log in as `root` and set a password.
 
-Perform the following steps:
+#. Verify you have a network connection to the Internet and configure proxy
+   settings if you're working behind a firewall.
 
-#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.
+#. Download a sample YAML configuration file. For example, if you want to 
+   install |CL| with a desktop GUI, you might want to use `live-desktop.yaml`_.
+   Or you can use the `live-server.yaml`_ if you want to install a non-GUI version
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: These links didn't work when I tested it. I recommend removing them. You already show the curl commands below for both the live-server and the live-desktop.  Those commands work fine.

File: source/get-started/install-configfile.rst [hunk 3]
```diff
@@ -15,67 +16,61 @@ machines.
...
-#. Boot up the USB thumb drive.
-#. Select :guilabel:`Clear Linux OS` from the menu.
-#. In the console window, log in as root and set a password.
-#. Verify you have a network connection to the Internet and configure proxy
-   settings if you're working behind a firewall.
-#. Download a :file:`live-server.yaml` template.
+        curl -O https://cdn.download.clearlinux.org/current/config/image/live-desktop.yaml
 
-   For example:
+    * Server:
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: Make sure that the `Server` block is left-aligned to the page.

File: source/get-started/install-configfile.rst [hunk 4]
```diff
@@ -15,67 +16,61 @@ machines.
...
 
-   .. code-block:: bash
+      .. code-block:: bash
 
-      curl -O https://download.clearlinux.org/releases/30010/clear/config/image/live-server.yaml
+         curl -O https://cdn.download.clearlinux.org/current/config/image/live-server.yaml
 
-#. Edit the template and change the settings as needed.
+#. Edit the YAML configuration file and change the settings as needed.
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: Please give the YAML file a figure number. Then refer to that figure as "...shown below in Figure 1."

File: source/get-started/install-configfile.rst [hunk 5]
```diff
@@ -131,54 +125,51 @@ Perform the following steps:
 
 .. _install-configfile-yaml-end:
 
-Start the installation with the command:
+#. Start the unattended installation using the `--config` option.
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: This step need to be left-aligned to the page. It shouldn't restart numbering at 1. 

Also, I recommend adding a line to indicate that the user should complete installation.

File: source/get-started/install-configfile.rst [hunk 6]
```diff
@@ -131,54 +125,51 @@ Perform the following steps:
...
-   clr-installer --config live-server.yaml
+      clr-installer --config live-server.yaml
 
-Example 2
-=========
+Example 2: Replicate a previous installation
+********************************************
 
-This method uses a saved configuration file from a previous installation,
+This example uses a saved configuration file from a previous installation,
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: Please make a direct link to this document: 
"... previous installation, as shown in : https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html#save-configuration-settings

File: source/get-started/install-configfile.rst [hunk 7]
```diff
@@ -15,67 +16,61 @@ machines.
 Prerequisites
 *************
 
-Ensure that your target system supports the installation:
+For installation onto bare metal, ensure that your target system 
+supports the installation:
 
 * :ref:`system-requirements`
 * :ref:`compatibility-check`
+* :ref:`bootable-usb` of the Live Server image
```
Comment (mvincerx) [inline source/get-started/install-configfile.rst]: I recommend revising this bullet:
`:ref: bootable-usb` of the  `:ref: bare-metal-install-server` .  

Add a note below to explain  to the user that it's only possible to create yaml files with the live-server version. I just want this to be very explicit.

File: source/get-started/install-configfile.rst (hunk 1 above)
Comment (bktan8) [reply source/get-started/install-configfile.rst]: I meant two.  fixed.

File: source/get-started/install-configfile.rst (hunk 2 above)
Comment (bktan8) [reply source/get-started/install-configfile.rst]: fixed.

File: source/get-started/install-configfile.rst (hunk 4 above)
Comment (bktan8) [reply source/get-started/install-configfile.rst]: It's a code-block.  Should we use a "Figure"?

File: source/get-started/install-configfile.rst (hunk 6 above)
Comment (bktan8) [reply source/get-started/install-configfile.rst]: it's not necessary to link this document.  clr-installer saves the YAML of an installed system to /root regardless.

File: source/get-started/install-configfile.rst (hunk 7 above)
Comment (bktan8) [reply source/get-started/install-configfile.rst]: That's not true.  The live-desktop version also saves the YAML automatically after installation is completed.

Comment (mvincerx) [reply source/get-started/install-configfile.rst]: Oh, yeah!. You're right.  We never documented that in the live-desktop installation guide. I think we should add a note there too.

**Pull request merged**